### PR TITLE
remove and add function implemented

### PR DESCRIPTION
**Pull request by olawalecoder**
Creating a new JavaScript file for new functionality.
Implementing a function for adding a new task (add a new element to the array).
Implementing a function for deleting a task (remove an element from the array).
Implementing a function for editing task descriptions and so on.

### CODE REVIEW

File: src/Todolist.js [hunk 1]
```diff
@@ -12,20 +12,22 @@ export default class Todolist {
...
+  static remove = (itemsToDelete, clear = false) => {
+    let todoList = StorageManager.getData();
+    if (!clear) {
+      itemsToDelete.forEach((index) => {
+        todoList = todoList.filter((todo) => todo.index !== Number(index));
+      });
+    } else {
+      todoList = todoList.filter((todo) => todo.completed !== true);
+    }
     Todolist.updateToDoItemIndex(todoList);
```
Comment (dasileker) [inline src/Todolist.js:24]: - [ ] Nice job on implementing the required codes however the remove condition is not working.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @olawalecoder ,** 

### **Great work on making the changes 👏** 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## **Suggested changes**
_Check the comments under the review._


- **Nice work mate, but there are multiple things to solve right now the delete method and the update method are not functioning + the trash icon is not rendered also.**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/Todolist.js (hunk 1 above)
Comment (olawalecoder) [reply src/Todolist.js:24]: Thanks I will work on it

**Comment by olawalecoder** [comment]
Hi, this is my last review chance. Please let me know via slack if there is any change you will like me to do. I will be on standby thanks

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### ** Hi @olawalecoder ** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html [hunk 2]
```diff
@@ -1,46 +1,29 @@
...
+        <div class="clear">
+            <a>Clear All</a>
         </div>
-      </li>
-    </ul>
-  </body>
-</html>
+    </div>
+</body>
+
```
[the remark this answers is not in the document]
Comment (olawalecoder) [reply src/index.html:28]: Thank you. I will implement all changes

**Pull request merged**